src/reducers/account_details.js

```diff
@@ -0,0 +1,20 @@
+import { TOGGLE_EDIT_MODE } from '../actions/account_details'
+
+const initialState = {
```

Comment:
I prefer the use of upper snake case for consts (`INITIAL_STATE`) - it was not clear to me how this was being used without reading the code below